src/test/java/eu/europeana/metis/sandbox/service/workflow/HarvestServiceImplTest.java

```diff
@@ -40,21 +41,22 @@
 import org.springframework.test.context.junit.jupiter.SpringExtension;
 import org.zeroturnaround.zip.ZipException;
 
-
 @ExtendWith(SpringExtension.class)
 public class HarvestServiceImplTest {
 
   private final TestUtils testUtils = new TestUtils();
 
-  private static HttpHarvester httpHarvester = spy(HarvesterFactory.createHttpHarvester());
+  private static final HttpHarvester httpHarvester = spy(HarvesterFactory.createHttpHarvester());
 
-  private static OaiHarvester oaiHarvester = mock(OaiHarvester.class);
+  private static final OaiHarvester oaiHarvester = mock(OaiHarvester.class);
 
-  private static HarvestServiceImpl harvestService = new HarvestServiceImpl(httpHarvester,
+  private static final HarvestServiceImpl harvestService = new HarvestServiceImpl(httpHarvester,
       oaiHarvester);
 
   @Test
-  void harvestServiceFromURL_ExpectSuccess() throws IOException {
+  void harvestServiceFromURL_noRecordLimit_ExpectSuccess() throws IOException {
```

Comment:
What do you mean noRecordLimit? 
There is a limit we just do not reach it, is that what is meant here?